geomstats/hypersphere.py

```diff
@@ -135,6 +135,24 @@ def random_uniform(self, n_samples=1):
 
         return point
 
+    def random_uniform_unbounded(self, n_samples=1):
```

Comment:
How about merging this into the function `random_uniform` as:
```
random_uniform(self, bound=None, n_samples=1)
```
and distinguishing the case when there is no bound (`bound` is None) and when there is a bound?